### DISCUSSION TITLE
Equivalent Orientations (Cubic, Orthotropic)

### DISCUSSION THREAD

**Discussion by bnmcbride** [mtex-toolbox/mtex#757] [Ask Anything]
I'm looking for a way to get equivalent texture components for a m-3m crystal symmetry with an orthorhombic sample symmetry. I'd like to have these displayed in the first 90x90x90 subspace. 

From my understanding "symmetrize(ori)" will list all equivalent orientations in ODF-space and "ori.project2EulerFR" will give the single component in the fundamental region, but how can I expand this to all equivalent orientations within the 90x90x90 subspace?

It seems that plotSection(ori) will plot these values....is there another way to output these values in the Command Window?

**Comment by kilir** [comment]
Hi,
maybe you need to rephrase a little bit what you like to achieve. You have an orientation and you would want to find all equivalent orientations which are all limited in Euler angles to 90x90x90?
Cheers,
Rüdiger
